views/candidates/showSingleCandidate.ejs

```diff
@@ -23,9 +23,8 @@
 								<p class="text-muted font-size-sm">
 									<%= card.current_location %>
 								</p>
-								<% if (!user.candidateProfile || !user.candidateProfile.equals(card._id)) { %>
-									<%- include('../partials/matches/connect.ejs') %>
-								<% } %>
+								<% if (!user.candidateProfile || !user.candidateProfile.equals(card._id)) { %> <%-
+								include('../partials/connect.ejs', {id: card._id, jobs: jobs }) %> <% } %>
```

Comment:
just wondering, if you decide to move it upper, will it work if we delete the percentage signs around } this curly bracket and put it after the closing brackets for include?
{id: card._id, jobs: jobs }) **}** %>

Reply:
just tested this out and doesn't work! It's because the opening tag for the include is different (<%- with the dash) than the normal one :)